component/firecracker-base/orchestrate-install.sh

```diff
@@ -198,18 +205,34 @@ execute_cleanup() {
         *)
             echo "Error: Something went wrong during cleanup, OS_VARIANT set to: $OS_VARIANT" && exit 1
         ;;
-    esac  
+    esac
 
     rm -Rf /tmp/firecracker-install/*
-    
+
+}
+
+prepare_jailers() {
+  if test -f "./prepare_jailer.sh"; then
+    ITERATIONS="${1:-100}" # Default to 100 jails
```

Comment:
non-blocking: For no other purpose than consistency: I suggest we make the default jail number the same as what veritech is currently hardcoded to (5000) at the minute.

Reply:
I don't want to do this until we get the CoW bits implemented. 5000 jails and 1gb images will be a bad time unless the disks are big enough

Reply:
Alrighty, can we also drop the num in veritech server too in the meantime?
https://github.com/systeminit/si/blob/main/lib/deadpool-cyclone/src/instance/cyclone/local_uds.rs#L700